rllib/tuned_examples/sac/pendulum-sac.yaml

```diff
@@ -19,10 +20,10 @@ pendulum-sac:
           fcnet_hiddens: [256, 256]
         tau: 0.005
         target_entropy: auto
-        no_done_at_end: true
+        no_done_at_end: True
         n_step: 3
         rollout_fragment_length: 1
-        prioritized_replay: true
+        prioritized_replay: False
```

Comment:
is this necessary? harder to say where the difference comes from if we change multiple variables in a single PR.
I am hoping using prioritized_replay would help learning, but maybe slowing things down a bit?

Reply:
I can change it back, but it's possible that this hurts and doesn't help. I'll remove it and ablate the change, then post the learning curve in the channel.

Reply:
cool, I am really curious to see the learning curve. if we need this, we can keep it no problem, but would love to have a bit better understanding.